queries/cmake/locals.scm

```diff
@@ -0,0 +1,11 @@
+(normal_command 
+  (identifier) @_command
+  (argument_list
+    .  (argument) @definition.var)
+  (#any-of? @_command
+   "set" "SET"
+   "option" "OPTION"))
+
+(variable_ref) @reference
+
+(source_file) @scope
```

Comment:
Hmm, I don't think this file was intended to be here

Reply:
sorry. fixed